WORKSPACE

```diff
@@ -101,6 +99,15 @@ filegroup (
   ]),
   testonly = 1,
 )
+
+filegroup (
```

Comment:
Is this used?

Reply:
I guess it's not used now. Let's try to remove it.